src/riscv-integration.adoc

```diff
@@ -454,9 +415,9 @@ include::generated/csr_added_purecap_mode_u_table_body.adoc[]
 
```

Comment:
The table `ucsrnames-added` used to generate a list of all unprivileged CSRs that CHERI added --DDC was the only one. However, the table is now coming out empty.

Reply:
it's empty as we don't add any user mode CSRs in purecap mode as DDC is added in legacy mode
so we could remove the table completely